localization/gyro_odometer/README.md

```diff
@@ -28,4 +27,12 @@
 
 ## Assumptions / Known limits
 
-TBD.
+- [Assumption] The frame_id of input twist message must be set to base_link.
+
+- [Assumption] The covariance in the input messages must be properly assigned.
+
+- [Assumption] The angular velocity is set to zero if both the longitudinal vehicle velocity and the angular velocity around the yaw axis are sufficiently small. This is for suppression of the IMU angular velocity bias. Without this process, we misestimate the vehicle status when stationary.
+
+- [Limitation] The frequency of the output IMU message depends on the frequency of the input IMU message.
+
+- [Limitation] We cannot produce reliable values for the lateral and vertical velocities and also for the angular velocity around pitch and roll axes. Therefore we assign large values to the corresponding elements in the output covariance matrix.
```

Comment:
Sorry for making comment again. I forgot `the`

```
[Limitation] We cannot produce reliable values for the lateral and vertical velocities and the angular velocity around the pitch and roll axes. Therefore we assign large values to the corresponding elements in the output covariance matrix.
```

Reply:
I'm sorry, I forgot to fix https://github.com/autowarefoundation/autoware.universe/pull/412#discussion_r809830417.
I removed the roll and pitch from the text, they are not necessary.

Reply:
How about https://github.com/autowarefoundation/autoware.universe/pull/412/commits/74c6834b2fd39d16f2d7b9852c4b7c4b82dd0125 ?